clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -1085,6 +1085,68 @@ def CmpOp : CIR_Op<"cmp", [Pure, SameTypeOperands]> {
   let hasVerifier = 0;
 }
 
+//===----------------------------------------------------------------------===//
+// CheckedArithOp
+//===----------------------------------------------------------------------===//
+
+def CAK_Add : I32EnumAttrCase<"Add", 1, "add">;
+def CAK_Sub : I32EnumAttrCase<"Sub", 2, "sub">;
+def CAK_Mul : I32EnumAttrCase<"Mul", 3, "mul">;
```

Comment:
We can reuse a bit here. Instead of adding these, just add a new `BinOpCheckedKind` `I32EnumAttr` and use `BinOpKind_Add`, etc.

Reply:
Updated.